app/controllers/comment_controller.php

```diff
@@ -1,7 +1,71 @@
 <?php
 	class CommentController extends AppController
 	{
+		public function write()                        
+	    {
+	        $thread = Thread::get(Param::get('thread_id'));
+	        $threadId = Param::get('thread_id');
+	        $comment = new Comment;
+	        $page = Param::get('page_next', 'write');
+	        switch ($page) {
+	        case 'write':                    
+	            break;
+	        case 'write_end':                
+	            $comment->username = Param::get('username');
+	            $comment->body = Param::get('body');
```

Comment:
The case statement MUST be indented once from switch, and the break keyword (or other terminating keyword) MUST be indented at the same level as the case body.
```
switch ($expr) {
    case 1:
        break;
    case 2':
        ...
        break;
    default:
        ...
        break;
}
```